src/kusto/setup.py

```diff
@@ -9,9 +9,12 @@
 from codecs import open
 from setuptools import setup, find_packages
 
-# TODO: Confirm this is the right version number you want and it matches your
 # HISTORY.rst entry.
 VERSION = '0.1.0'

```

Comment:
Release will be triggered until VERSION is upgraded. Such at `0.1.1`, `0.2.0`

Reply:
upgraded to 0.1.1